ygen/directory.go

```diff
@@ -137,38 +137,47 @@ func GetOrderedDirectories(directory map[string]*Directory) ([]string, map[strin
 // of a Directory. The Field is specified as a name in order to guarantee its
 // existence before processing.
 func FindSchemaPath(parent *Directory, fieldName string, absolutePaths bool) ([]string, error) {
-	return findSchemaPath(parent, fieldName, false, absolutePaths)
+	schemaPaths, _, err := findSchemaPath(parent, fieldName, false, absolutePaths)
+	return schemaPaths, err
 }
 
 // findSchemaPath finds the relative or absolute schema path of a given field
 // of a Directory, or the shadowed field path (field duplicated and
 // deprioritized via compression) of a Directory. The Field is specified as a
 // name in order to guarantee its existence before processing.
 // NOTE: If shadowSchemaPaths is true, no error is returned if fieldName is not found.
```

Comment:
What is the 2nd return argument here that you added? It looks like the docstring here did not get updated?

Reply:
Done.